modules/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -133,15 +140,36 @@ export default class MeshLayer extends Layer {
         accessor: 'getPosition',
         update: this.calculateInstancePositions64xyLow
       },
-      instanceRotations: {
-        size: 3,
-        accessor: ['getYaw', 'getPitch', 'getRoll'],
-        update: this.calculateInstanceRotations
-      },
       instanceColors: {
         size: 4,
         accessor: 'getColor',
         defaultValue: [0, 0, 0, 255]
+      },
+      instanceModelMatrix: {
+        size: 16,
+        shaderAttributes: {
+          instanceModelMatrixColumn1: {
+            size: 4,
+            stride: 64,
+            offset: 0
+          },
+          instanceModelMatrixColumn2: {
+            size: 4,
+            stride: 64,
+            offset: 16
+          },
+          instanceModelMatrixColumn3: {
+            size: 4,
+            stride: 64,
+            offset: 32
+          },
+          instanceModelMatrixColumn4: {
+            size: 4,
+            stride: 64,
+            offset: 48
+          }
+        },
+        update: this.calculateInstanceXform
```

Comment:
Nit: Use same name for same thing: `this.calculateInstanceXform` => `this.calculateInstanceModelMatrix`